standalone/main.cpp

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) 2020 Pilz GmbH & Co. KG
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+#include <string>
+#include <iostream>
+
+#include "psen_scan_v2/default_parameters.h"
+#include "psen_scan_v2/laserscan.h"
+#include "psen_scan_v2/scanner_configuration.h"
+#include "psen_scan_v2/scanner_config_builder.h"
+#include "psen_scan_v2/scanner_v2.h"
+#include "psen_scan_v2/scan_range.h"
```

Comment:
Please create a convenience include, to simplify the usage to only one user include.

Reply:
It is defined inside [scan_range.h](https://github.com/PilzDE/psen_scan_v2/blob/ef117b0865eaf8ac01db90e7658ac8a1502d62af/standalone/include/psen_scan_v2/scan_range.h#L87)

Maybe @martiniil or @agutenkunst knows more on why the `ScanRange` was implemented in this templated format.

Reply:
done